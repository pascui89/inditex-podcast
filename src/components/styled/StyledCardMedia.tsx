import {
    CardMedia,
    styled,
  } from '@mui/material';

export const StyledCardMedia = styled(CardMedia)`
    height: 150px;
    width: 150px;
`;