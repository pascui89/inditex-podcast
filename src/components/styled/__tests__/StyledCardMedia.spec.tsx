import React from 'react';
import { render } from '@testing-library/react';
import { StyledCardMedia } from '../StyledCardMedia';
import '@testing-library/jest-dom/extend-expect';

global.React = React;

describe('StyledCardMedia', () => {

  it('should render', () => {
    const { container } = render(<StyledCardMedia/>);
    expect(container).toMatchSnapshot();
  });
});
