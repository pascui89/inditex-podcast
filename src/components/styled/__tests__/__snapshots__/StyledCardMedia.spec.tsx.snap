// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledCardMedia should render 1`] = `
<div>
  <div
    class="MuiCardMedia-root css-132720y-MuiCardMedia-root"
  />
</div>
`;
